package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cloudant/couchbackup",
-  "version": "2.6.3-SNAPSHOT",
+  "version": "2.6.2-SNAPSHOT",
```

Comment:
```suggestion
  "version": "2.6.3-SNAPSHOT",
```

Reply:
Thanks should have spotted that in the rebase. Fixed in 528e900241f3c2d1c60a0d61fadf63faffbf7a6f (as I needed the package-lock.json too)